Resources/Prototypes/Entities/Objects/Consumable/Food/snacks.yml

```
@@ -189,6 +189,73 @@
     color: white
   - type: Food
     trash: FoodPacketSyndiTrash
+    
+- type: entity
+  name: chow mein
+  parent: FoodSnackBase
+  id: FoodSnackChowMein
+  description: A salty fried noodle snack. Looks like they forgot the vegetables.
+  components:
+  - type: Sprite
+    state: chinese1
+  - type: Item
+    color: red
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 30 # Room for extra condiments
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 10
+        - ReagentId: Soysauce
+          Quantity: 2
+  - type: Food
+    trash: FoodPacketChowMeinTrash
+          
+- type: entity
+  name: dan dan noodles
+  parent: FoodSnackBase
+  id: FoodSnackDanDanNoodles
+  description: A spicy Sichuan noodle snack. The chilli oil slick pools on top.
+  components:
+  - type: Sprite
+    state: chinese2
+  - type: Item
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 30 # Room for extra condiments
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 10
+        - ReagentId: CapsaicinOil
+          Quantity: 2
+        - ReagentId: Soysauce
+          Quantity: 2
+  - type: Food
+    trash: FoodPacketDanDanTrash
+
+- type: entity
+  name: fortune cookie
+  parent: FoodSnackBase
+  id: FoodSnackCookieFortune
+  description: A boring cardboard tasting snap with a fortune inside. Surprise! You're boring too.
```

```suggestion
  description: A boring cardboard tasting snack with a fortune inside. Surprise! You're boring too.
```
Or maybe change the description to one of the SS13 ones? Fortune cookies don't really taste like cardboard.

---

You'd be surprised how many cheap ones i've had that do in fact taste more like cardboard than anything.

---

Yeah this is off personal experience. Any pre packaged one I've ever tasted did not taste edible. Usually from takeaway.